docs/fr/docs/advanced/path-operation-advanced-configuration.md

```diff
@@ -1,99 +1,99 @@
 
-It won't show up in the documentation, but other tools (such as Sphinx) will be able to use the rest.
+Il n'apparaîtra pas dans la documentation, mais d'autres outils (tels que Sphinx) pourront utiliser le reste.
 
 ```Python hl_lines="19-29"
 {!../../../docs_src/path_operation_advanced_configuration/tutorial004.py!}
 ```
 
-## Additional Responses
+## Réponses supplémentaires
 
-You probably have seen how to declare the `response_model` and `status_code` for a *path operation*.
+Vous avez probablement vu comment déclarer le `response_model` et le `status_code` pour une *opération de chemin*.
 
-That defines the metadata about the main response of a *path operation*.
+Cela définit les métadonnées sur la réponse principale d'une * opération de chemin *.
 
-You can also declare additional responses with their models, status codes, etc.
+Vous pouvez également déclarer des réponses supplémentaires avec leurs modèles, codes de statut, etc.
 
-There's a whole chapter here in the documentation about it, you can read it at [Additional Responses in OpenAPI](./additional-responses.md){.internal-link target=_blank}.
+Il y a un chapitre entier ici dans la documentation à ce sujet, vous pouvez le lire sur [Réponses supplémentaires dans OpenAPI](./additional-responses.md){.internal-link target=_blank}.
 
-## OpenAPI Extra
+## OpenAPI supplémentaire
 
-When you declare a *path operation* in your application, **FastAPI** automatically generates the relevant metadata about that *path operation* to be included in the OpenAPI schema.
+Lorsque vous déclarez un *chemin* dans votre application, **FastAPI** génère automatiquement les métadonnées concernant ce * chemin* à inclure dans le schéma OpenAPI.
 
-!!! note "Technical details"
-    In the OpenAPI specification it is called the <a href="https://github.com/OAI/OpenAPI-Specification/blob/main/versions/3.0.3.md#operation-object" class="external-link" target="_blank">Operation Object</a>.
+!!! note "Détails techniques"
+    La spécification OpenAPI appelle ces métaonnées des <a href="https://github.com/OAI/OpenAPI-Specification/blob/main/versions/3.0.3.md#operation-object" class="external-link" target="_blank">Objets d'opération</a>.
 
-It has all the information about the *path operation* and is used to generate the automatic documentation.
+Il contient toutes les informations sur le *chemin* et est utilisé pour générer automatiquement la documentation.
 
-It includes the `tags`, `parameters`, `requestBody`, `responses`, etc.
+Il inclut les `tags`, `parameters`, `requestBody`, `responses`, etc.
 
-This *path operation*-specific OpenAPI schema is normally generated automatically by **FastAPI**, but you can also extend it.
+Ce schéma OpenAPI spécifique aux *operations* est normalement généré automatiquement par **FastAPI**, mais vous pouvez également l'étendre.
 
 !!! tip
-    This is a low level extension point.
+    Il s'agit d'extension de bas niveau.
```

Comment:
hummmm I think this line is confusing, I would i be in favor of :fire: it ! 
Wdyt ?